### DISCUSSION TITLE
How to use output file tracing in custom-server-example?

### DISCUSSION THREAD

**Discussion by WhiteYin** [vercel/next.js#41652] [Help]
Firstly, I need to use a custom server to implement api proxy, like [the official example](https://github.com/vercel/next.js/tree/main/examples/custom-server-express).
Then, I use Docker to deploy the app.
To reduce the size of image, I can use `output: 'standalone'` in `next.config.js` as [output file tracing](https://nextjs.org/docs/advanced-features/output-file-tracing).
But I found that my custom server and related dependencies was not copied to `.next/standalone`.

How to solve it?

**Comment by samueldusek** [comment]
Hi @WhiteYin ,

did you manage to solve this issue?

**Reply by WhiteYin** [reply]
sorry, I don't. Maybe I can write a script with nodejs to manually install or copy these dependencies

**Comment by pluppplupp** [comment]
Did you ever find a solution on this? Same problem here, still..
